dirsrvtests/tests/suites/fourwaymmr/fourwaymmr_test.py

```diff
@@ -439,13 +450,16 @@ def test_replica_backup_and_restore(topo_m4):
         5. Should success
     """
     # Testing bug #830335: Taking a replica backup and Restore on M1 after deleting few entries from M1 nad M2
+    # Make sure to update S1 to avoid useless replication delay
```

Comment:
I would not change the supplier in that test. : We do not have lot of 4 supplier tests and IMHO it is a good idea to have a test in which the changes are not issued on supplier 1 .
IMHO we should rather keep creating the user on supplier 3 but wait 
repl.wait_for_replication(topo_m4.ms["supplier3"], topo_m4.ms["supplier1"]) before waiting for supplier1 -> supplier2